README.md

```diff
@@ -469,35 +465,43 @@ Fully customisable.  |
 :star2: | Name | Description | 🌍
 --- | --- | --- | ---
 2 | [@mkatircioglu](https://github.com/mkatircioglu)/[**dotfiles**](https://github.com/mkatircioglu/dotfiles) |  |
-### Crystal #
```

Comment:
@lacivert The `package.json` file is used when generating the `README.md` file. The `README.md` is generated.